.ci/load/scripts/locustfile.py

```diff
@@ -0,0 +1,13 @@
+from locust import HttpUser, task, between
+
+
+class QuickstartUser(HttpUser):
+    wait_time = between(1, 2)
```

Comment:
Can we make it executing requests at a fixed rate in order to avoid coordinated omission problems?

Reply:
Most certainly! I will make a change to the PR to modify the defaults.

(Note that this pipeline also supports passing in your own Locust configuration file so you can design a test however you see fit. Obviously, I started with something super-basic but I'm thinking that we could also design "scenarios" which pre-package various load test configurations as well, but that's more of a long-term roadmap item.)

Reply:
Updated. Some additional reading on this for folks who might be interested:

https://github.com/locustio/locust/issues/646
https://github.com/locustio/locust/pull/1118

Down the road, there's no reason we couldn't explore supporting different types of load-generation tools if we really wanted to, but IMHO we should see how far this one gets us for the time being. I'm open to swapping it out if it doesn't meet our needs, though.